<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-BLPVR62" failures="1" tests="4" name="seleniumWebdriver.TC006" time="6.553" errors="0" timestamp="2024-11-30T22:26:54 IST" skipped="0">
  <testcase classname="seleniumWebdriver.TC006" name="testOpenApplication" time="6.173"/>
  <system-out/>
  <testcase classname="seleniumWebdriver.TC006" name="testUsername" time="0.030">
    <failure message="expected [] but found [[[ChromeDriver: chrome on windows (b4560a1426277523dd9d208ff8ae15b9)] -&gt; xpath: //input[@id=&#039;email&#039;]]]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [] but found [[[ChromeDriver: chrome on windows (b4560a1426277523dd9d208ff8ae15b9)] -> xpath: //input[@id='email']]]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:617)
at seleniumWebdriver.TC006.testUsername(TC006.java:28)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- testUsername -->
  <system-out/>
  <testcase classname="seleniumWebdriver.TC006" name="testPassword" time="0.225"/>
  <system-out/>
  <testcase classname="seleniumWebdriver.TC006" name="testlogin" time="0.125"/>
  <system-out/>
</testsuite> <!-- seleniumWebdriver.TC006 -->
